nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -1150,7 +1164,10 @@ def write_sorting(
     skip_properties: Optional[List[str]] = None,
     skip_features: Optional[List[str]] = None,
     use_times: bool = True,
-    metadata: dict = None,
+    metadata: Optional[dict] = None,
+    write_as: str = "units",
+    units_name: str = "units",
```

Comment:
Can you add a check to make sure that if `write_as=="units"` then `units_name` must also be `"units"`

Reply:
Added!